models/articles.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT articles.*, COUNT(comments.comment_id) AS comment_count
+      FROM articles LEFT JOIN comments
+      ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id ORDER BY articles.created_at DESC;`
+    )
+    .then(({ rows }) => {
+      const deleteUnuseItem = rows.map((each) => {
+        delete each.body;
+        return each;
+      });
```

Comment:
this works but we could select everything apart from articles.body in your SQL query

Reply:
I will take a look at the syntax! thanks for your advice:))